src/web/modules/stripe/basic/basic.http.ts

```diff
@@ -98,7 +101,24 @@ const submitAccountCreate = async function submitAccountCreate(
   }
 }
 
+const logStripeBalance = async function logStripeBalance(
+  req: Request,
+  res: Response
+): Promise<void> {
+  if (!STRIPE_ACCOUNT_API_KEY) {
+    throw new CustomValidationError('Stripe API Key was not configured for this Toolbox instance')
+  }
+  const balance = await stripeClient.getStripeApi().balance.retrieve()
```

Comment:
It wasn't clear to me that we're using a shared Stripe client in this case because we set up a variable for the API key, it was confusing that we then didn't use it.

I think to achieve the same result but avoid making it look like we're going to use the API key we can just directly check the environment

```js 
if (!process.env.STRIPE_ACCOUNT_API_KEY) {
  // ...
}
```